ingestion/src/metadata/ingestion/source/redshift_usage.py

```diff
@@ -45,26 +51,34 @@ class RedshiftUsageSource(Source[TableQuery]):
     SERVICE_TYPE = DatabaseServiceType.Redshift.value
     DEFAULT_CLUSTER_SOURCE = "CURRENT_DATABASE()"
 
-    def __init__(self, config, metadata_config):
+    def __init__(
+        self, config: WorkflowSource, metadata_config: OpenMetadataServerConfig
+    ):
         super().__init__()
         self.config = config
+        self.source_connection = config.serviceConnection.__root__.config
```

Comment:
let's not mix up names and create two different variables based on use cases.
service_connection for config.serviceConnection.__root__.config
and source_config for config.sourceConfig.config.

And use them in other places as well